### ISSUE TITLE
`Import` has completions for files but not for modules

### ISSUE THREAD

**Issue by JasonGross** [cpitclaudel/company-coq#113]
![image](https://cloud.githubusercontent.com/assets/396076/15021754/8bac35a2-11f6-11e6-8445-8b7fe801e5eb.png)

**Comment by cpitclaudel** [comment]
Yup, it's a known-ish issue. Is there a way to enumerate all modules?

**Comment by JasonGross** [comment]
None that I know of, but it's probably an easy patch to write (look at `<global environment>.env_modules.module_path`, I think?).

**Comment by cpitclaudel** [comment]
My last patch to get more completions is still in limbo, so I think I'll hold off on that :)
